src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewcollections/ViewCollectionStyles.kt

```diff
@@ -0,0 +1,39 @@
+package org.wycliffeassociates.otter.jvm.app.ui.viewcollections
+
+import javafx.geometry.Pos
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.viewcontent.ViewContentStyles
+import tornadofx.*
+
+class ViewCollectionStyles: Stylesheet() {
```

Comment:
What are ViewCollections?

Reply:
it's a collections grid. i didn't choose the name very well